README.md

```diff
@@ -34,32 +39,52 @@ fn main(){
 To get all the files with a specific extension in a directory, use:
 
 ```rust
-use rust_search::Search;
+use rust_search::SearchBuilder;
 
-Search::new("/path/to/directory", None, Some(".fileExtension"), Some(1));
+let _ = SearchBuilder::default()
+    .location("/path/to/directory")
+    .ext("file_extension")
+    .depth(1)
+    .build();
 ```
 
 To get all the files in a directory, use:
 
 ```rust
-use rust_search::Search;
+use rust_search::SearchBuilder;
 
-Search::new("/path/to/directory", None, None, Some(1));
+let _ = SearchBuilder::default()
```

Comment:
someone might expect this to print the values... Lets store it in a variable and show that it's a vector.